content/_data/indexpage/carousel.yml

```diff
@@ -1,16 +1,17 @@
-# cdCon
-- :href: https://events.linuxfoundation.org/cdcon/
-  :title: cdCon
-  :intro: Join Jenkins at cdCon on June 7-8, 2022! 
-    The event is focused on improving the world's capacity to deliver software with security and speed. 
-    Become part of the conversation that drives continuous delivery by meeting peers, 
-    sharing ideas and talking to industry leaders on all things software delivery and DevOps.
+# DevOps World 2022
+- :href: https://reg.devopsworld.com/flow/cloudbees/devopsworld22/Landing/page/welcome
+  :title: Join Jenkins at DevOps World!
+  :intro:  
+    Get ready for the DevOps remix at the Live and In-Person DevOps World 2022 on September 27-29 — the largest 
+    global gathering of DevOps thought leaders, practitioners, and contributors, and all those dedicated to 
```

Comment:
global gathering of DevOps thought leaders, practitioners, Jenkins contributors, and all those dedicated to